src/client.rs

```diff
@@ -588,7 +589,8 @@ impl Client {
 #[serde(rename_all = "camelCase")]
 pub struct ClientStats {
     pub database_size: usize,
-    pub last_update: Option<String>,
+    #[serde(with = "time::serde::rfc3339::option")]
+    pub last_update: Option<OffsetDateTime>,
```

Comment:
Should I be worried because no tests were changed and, we still have the CI passing? 😅

Reply:
I totally get your concern ahah!

We actually have only [one test](https://github.com/meilisearch/meilisearch-rust/blob/main/src/client.rs#L216-L223) on this route:
```rust
use meilisearch_sdk::{client::*, indexes::*};
futures::executor::block_on(async move {
  let client = Client::new("http://localhost:7700", "masterKey");
  let stats = client.get_stats().await.unwrap();
});
```
And the `unwrap` right there ensure we were able to parse the new date format:  `client.get_stats().await.unwrap();`

------

On another note I'm wondering if we could add more tests though, but we can't test a lot of things because we don't know what the `database_size` is supposed to be, we don't really know the time of the `last_update` or what indexes are in the `Client` currently :thinking: 
```rust
pub struct ClientStats {
    pub database_size: usize,
    pub last_update: Option<String>,
    pub indexes: HashMap<String, IndexStats>,
}
```
One test we could add though would be to create an index and ensure we find it later in the stats + ensure the time of the `last_update` >= the time we sent our update but that feel like testing meilisearch more than meilisearch-rust, what do you think?

Reply:
I totally agree with that idea! Or maybe if it is possible you could create a mock of the API response, adding a specific value in the `database_size` and `last_update` it will make it easier to test after that. But of course, this mock need only is used in the raw response of the API, to ensure the crate will be able to parse it accordingly. 

I think this way you could have the whole TDD flow :)

Reply:
Creating a mock would be the best thing, but I don't have the time currently, sadly :unamused: 
We could open an issue, though, I think [faux](https://docs.rs/faux/latest/faux/) would be really suited for this library :+1: